### PR TITLE
Fix code coverage report (closes #437).

### LINKED ISSUE #437: codecov.io for master branch reports 0.00% and "File path not found".
Please have a look at https://codecov.io/github/cloudflare/cfssl?branch=master
Issue is described as in title. 
Is there a problem in travis.yaml or test.sh file?

### PR DESCRIPTION
**Pull request by kisom**

### CODE REVIEW

**Comment by kisom** [comment]
Waiting for Travis tests to verify that this works.

**Comment by kisom** [comment]
Looks like [it's fixed now](https://codecov.io/github/cloudflare/cfssl?pr=452).

File: test.sh [hunk 1]
```diff
@@ -7,23 +8,29 @@ PACKAGES=$(go list ./... | grep -v /vendor/)
...
 for package in $PACKAGES
 do
-        fgt golint ${package##github.com/cloudflare/cfssl/}/*.go
+    fgt golint ${package##github.com/cloudflare/cfssl/}/*.go
 done
 
 # Build and install cfssl executable in PATH
 go install github.com/cloudflare/cfssl/cmd/cfssl
-
 go test $BUILD_FLAGS $PACKAGES
```
Comment (andrewbuss) [inline test.sh]: We seem to be running tests twice here. I moved $BUILD_FLAGS into the `go test` command on line 33 and removed this line without (apparent) issue. Is there a reason to keep this here?

Comment (andrewbuss) [inline test.sh]: Test duration doubled with this commit: 

https://github.com/andrewbuss/cfssl/commit/27edf77db2d43126cecdeae13a56684b7d0623ff#diff-1b531c5e405e0bc95890d52cf2f2f5b5L27 unduplicates the test

**Pull request merged**